proto/mantrachain/sanction/v1/genesis.proto

```
@@ -0,0 +1,18 @@
+syntax = "proto3";
+package mantrachain.sanction.v1;
+
+import "amino/amino.proto";
```

Buf (1.47.2)

import "amino/amino.proto": file does not exist

(COMPILE)